docs/algorithms/sig/mayo.yml

```diff
@@ -0,0 +1,112 @@
+name: MAYO
+type: signature
+principal-submitters:
+- Ward Beullens
+- Fabio Campos
+- Sofía Celi
+- Basil Hess
+- Matthias J. Kannwischer
+crypto-assumption: multivariable quadratic equations, oil and vinegar
+website: https://pqmayo.org
+nist-round: 1
+spec-version: https://eprint.iacr.org/2023/1683
+primary-upstream:
+  source: https://github.com/PQCMayo/MAYO-C/commit/fc9079fb5ac5cd4af98e3e0f094a0a3cf2a01499
+    with copy_from_upstream patches
+  spdx-license-identifier: Apache-2.0
+parameter-sets:
+- name: MAYO_1
+  claimed-nist-level: 1
+  claimed-security: EUF-CMA
+  length-public-key: 1168
+  length-secret-key: 24
+  length-signature: 321
+  implementations-switch-on-runtime-cpu-features: true
+  implementations:
+  - upstream: primary-upstream
+    upstream-id: opt
+    supported-platforms: all
+    common-crypto:
+    - SHA3: liboqs
+    - AES: liboqs
+    no-secret-dependent-branching-claimed: true
+    no-secret-dependent-branching-checked-by-valgrind: true
+    large-stack-usage: false
+  - upstream: primary-upstream
+    upstream-id: avx2
+    supported-platforms:
+    - architecture: x86_64
+      operating_systems:
+      - Darwin
+      - Linux
+      required_flags:
+      - avx2
+    common-crypto:
+    - SHA3: liboqs
+    - AES: liboqs
+    no-secret-dependent-branching-claimed: true
+    no-secret-dependent-branching-checked-by-valgrind: true
+    large-stack-usage: false
+- name: MAYO_2
+  claimed-nist-level: 1
+  claimed-security: EUF-CMA
+  length-public-key: 5488
+  length-secret-key: 24
+  length-signature: 180
+  implementations-switch-on-runtime-cpu-features: true
+  implementations:
+  - upstream: primary-upstream
+    upstream-id: opt
+    supported-platforms: all
+    common-crypto:
+    - SHA3: liboqs
+    - AES: liboqs
+    no-secret-dependent-branching-claimed: true
+    no-secret-dependent-branching-checked-by-valgrind: true
+    large-stack-usage: false
+  - upstream: primary-upstream
+    upstream-id: avx2
+    supported-platforms:
+    - architecture: x86_64
+      operating_systems:
+      - Darwin
+      - Linux
+      required_flags:
+      - avx2
+    common-crypto:
+    - SHA3: liboqs
+    - AES: liboqs
+    no-secret-dependent-branching-claimed: true
+    no-secret-dependent-branching-checked-by-valgrind: true
+    large-stack-usage: false
+- name: MAYO_3
```

Comment:
MAYO_5 unfortunately has a bigger stack usage (yet) and and won't fit the AFAIK ~500K stack available in secondary threads used in one of the OQS tests.

Reply:
There's plenty more algorithms failing (some many more than) single OQS tests and they're still included.

I'd thus suggest including MAYO_5, too. This would also 
- eliminate the need to keep answering the same question over and over again;
- allow us(ers) to experiment with the algorithm in integrations (which still is the main raison d'etre of OQS);
- allow us to save on PR(review)s

Reply:
I agree @baentsch, I'll add it and mark it with `large-stack-usage: true`.